server/src/routes/api/v1/sitesRouter.js

```diff
@@ -1,26 +1,60 @@
 import express from "express";
+import uploadImage from "../../../services/uploadImage.js";
+import cleanUserInput from "../../../services/cleanUserInput.js";
 import { Site } from "../../../models/index.js";
+import { ValidationError } from "objection";
+import SiteSerializer from "../../../serializers/SiteSerializer.js"
 
 const sitesRouter = new express.Router();
 
 sitesRouter.get("/", async (req, res) => {
     try {
         const sites = await Site.query();
-        return res.status(200).json({ siteList: sites });
+        
+        const serializedSites = sites.map(site => {
+            return SiteSerializer.getSummary(site)
```

Comment:
since there are 2 separate site serializer methods, consider updating `getSummary` to accept an array of sites and perform this map iteration inside the method to move logic out of the endpoint